crates/query-engine/translation/tests/snapshots/tests__select_where_or_to_in_optimization.snap

```diff
@@ -0,0 +1,54 @@
+---
+source: crates/query-engine/translation/tests/tests.rs
+expression: result
+---
+SELECT
+  coalesce(json_agg(row_to_json("%2_universe")), '[]') AS "universe"
+FROM
+  (
+    SELECT
+      *
+    FROM
+      (
+        SELECT
+          coalesce(json_agg(row_to_json("%3_rows")), '[]') AS "rows"
+        FROM
+          (
+            SELECT
+              "%1_Album"."AlbumId" AS "AlbumId"
+            FROM
+              (
+                SELECT
+                  "%0_Album".*
+                FROM
+                  "public"."Album" AS "%0_Album"
+                WHERE
+                  (
+                    (
```

Comment:
Now the optimization doesn't drop the other "or" values, and can even group multiple columns (here we do both `Title` and `ArtistId`)